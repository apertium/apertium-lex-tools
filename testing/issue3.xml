<rules>
  <rule weight="2.0">
    <match lemma="ver" tags="vblex"/>
    <match lemma="X" tags="adv">
      <select lemma="X"/>
    </match>
  </rule>
</rules>
