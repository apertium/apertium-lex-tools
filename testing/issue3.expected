^version<vblex>/version<vblex>$ ^X<adv>/X<adv>/Y<adv>$

^verity<vblex>/verity<vblex>$ ^X<adv>/X<adv>/Y<adv>$

^avere<vblex>/avere<vblex>$ ^X<adv>/X<adv>/Y<adv>$

^devolver<vblex>/devolver<vblex>$ ^X<adv>/X<adv>/Y<adv>$

^ver<vblex>/ver<vblex>$ ^X<adv>/X<adv>$
